test/unit/math/prim/mat/fun/cholesky_factor_transform_test.cpp

```diff
@@ -75,9 +75,9 @@ TEST(ProbTransform,choleskyFactorConstrainError) {
 
   Matrix<double,Dynamic,1> x(3);
   x << 1, 2, 3;
-  EXPECT_THROW(cholesky_factor_constrain(x,9,9),std::domain_error);
+  EXPECT_THROW(cholesky_factor_constrain(x,9,9), std::invalid_argument);
   double lp = 0;
-  EXPECT_THROW(cholesky_factor_constrain(x,9,9,lp),std::domain_error);
+  EXPECT_THROW(cholesky_factor_constrain(x,9,9,lp),std::invalid_argument);
```

Comment:
Thanks. Happy to update it. 

We haven't actually been linting our test code. I don't think we'll ever get that down to 0 lint errors. 

> On Aug 6, 2016, at 5:22 PM, Krzysztof Sakrejda <notifications@github.com> wrote:
> 
> In test/unit/math/prim/mat/fun/cholesky_factor_transform_test.cpp:
> 
> >    double lp = 0;
> > -  EXPECT_THROW(cholesky_factor_constrain(x,9,9,lp),std::domain_error);
> > +  EXPECT_THROW(cholesky_factor_constrain(x,9,9,lp),std::invalid_argument);
> Just linting on you but every other line has a space here 😱
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>